empress/support_files/js/empress.js

```diff
@@ -2630,11 +2648,12 @@ define([
             cladeInfo.sY = sY;
             cladeInfo.totalAngle = totalAngle;
 
-            // create 15 triangles to approximate sector
-            var deltaAngle = totalAngle / 15;
-            cos = Math.cos(deltaAngle);
-            sin = Math.sin(deltaAngle);
-            for (var line = 0; line < 15; line++) {
+            // create triangles to approximate sector
+            var numSamples = this._numSampToApproximate(totalAngle);
+            var deltaAngle = totalAngle / numSamples;
+            cos = Math.cos(0);
+            sin = Math.sin(0);
```

Comment:
I doubt that these would be a bottleneck, but we may as well avoid extra work where possible. If there are a ton of collapsed clades this _might_ save some time.
```suggestion
            cos = 1; // Math.cos(0)
            sin = 0; // Math.sin(0)
```